modules/ROOT/pages/to-share-an-http-api.adoc

```diff
@@ -1,17 +1,17 @@
-= Share an HTTP API Asset
+= Publish an HTTP API Asset
```

Comment:
I would also change the name of the file `to-share-an-http-api.adoc` to `to-publish-an-http-api.adoc` (and replacing it in the places it is)

Reply:
Good call!